src/lambda/local/debugConfiguration.ts

```diff
@@ -3,19 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import * as os from 'os'
-import * as vscode from 'vscode'
-import { DRIVE_LETTER_REGEX } from '../../shared/codelens/codeLensUtils'
+import { nodeJsRuntimes } from '../../lambda/models/samLambdaRuntime'
+import { SamLaunchRequestArgs } from '../../shared/sam/debugger/samDebugSession'
+import { dotNetRuntimes, pythonRuntimes, RuntimeFamily } from '../models/samLambdaRuntime'
 
-const DOTNET_CORE_DEBUGGER_PATH = '/tmp/lambci_debug_files/vsdbg'
+export const DOTNET_CORE_DEBUGGER_PATH = '/tmp/lambci_debug_files/vsdbg'
 
-export interface DebugConfiguration extends vscode.DebugConfiguration {
-    readonly type: 'node' | 'python' | 'coreclr'
-    readonly request: 'attach'
-}
-
-export interface NodejsDebugConfiguration extends DebugConfiguration {
-    readonly type: 'node'
+export interface NodejsDebugConfiguration extends SamLaunchRequestArgs {
```

Comment:
i agree, this is a planned refactoring. But the first step was to put the shape in place, so that redundant fields in this ADT become obvious and easy to eliminate.